api/src/main/resources/Java/libraries/okhttp-gson/api.mustache

```diff
@@ -0,0 +1,277 @@
+{{!
+
+This is an AoU-specific extension of the default Swagger-codegen okhttp API implementation class.
+
+This file was originally copied from the swagger-codegen repo in December 2019; the swagger-codegen file
+was last modified on Oct 2017: https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/api.mustache
+
+AoU-specific additions have been clearly bookended with comments marked "Begin AoU-specific additions".
+
+}}
+{{>licenseInfo}}
+
+package {{package}};
+
+import {{invokerPackage}}.ApiCallback;
+import {{invokerPackage}}.ApiClient;
+import {{invokerPackage}}.ApiException;
+import {{invokerPackage}}.ApiResponse;
+import {{invokerPackage}}.Configuration;
+import {{invokerPackage}}.Pair;
+import {{invokerPackage}}.ProgressRequestBody;
+import {{invokerPackage}}.ProgressResponseBody;
+{{#performBeanValidation}}
+import {{invokerPackage}}.BeanValidationException;
+{{/performBeanValidation}}
+
+import com.google.gson.reflect.TypeToken;
+
+import java.io.IOException;
+
+{{#useBeanValidation}}
+import javax.validation.constraints.*;
+{{/useBeanValidation}}
+{{#performBeanValidation}}
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.ValidatorFactory;
+import javax.validation.executable.ExecutableValidator;
+import java.util.Set;
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+{{/performBeanValidation}}
+
+{{#imports}}import {{import}};
+{{/imports}}
+
+{{! Begin AoU-specific additions }}
+import io.opencensus.common.Scope;
+import io.opencensus.trace.AttributeValue;
+import io.opencensus.trace.Span;
+import io.opencensus.trace.Status;
+import io.opencensus.trace.Tracer;
+import io.opencensus.trace.Tracing;
+import java.lang.reflect.Type;
+{{! End AoU-specific additions }}
+{{^fullJavaUtil}}
+import java.util.ArrayList;
```

Comment:
I think this is fine, but was just observing that it looks like these imports won't be in the canonical order after generation. Totally not worth fixing.

Reply:
Acknowledged. As discussed offline, I'm inclined to leave as-is for now, until we get a better feel for exactly how to treat overridden Swagger templates such as this.